recipe/meta.yaml

```diff
@@ -11,16 +10,16 @@ source:
   sha256: 75b1c665699ec0f1ffce1ba3d776f7dfce802156f22e70a7b9c8f0b4d7e80f42
 
 build:
-  skip: true  # [py<36]
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # maturin isn't availabe on s390x
+  skip: true  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
```

Comment:
```suggestion
```
The C compiler is a run dependency of rust_win-64. (respectively, rust-gnu_win-64 depends on m2w64-toolchain)